pkgs/by-name/ha/harec/package.nix

```diff
@@ -3,29 +3,33 @@
 , fetchFromSourcehut
 , qbe
 , fetchgit
+, unstableGitUpdater
 }:
 let
   # harec needs the dbgfile and dbgloc features implemented up to this commit.
   # This can be dropped once 1.2 is released, for a possible release date see:
   # https://lists.sr.ht/~mpu/qbe/%3CZPkmHE9KLohoEohE%40cloudsdale.the-delta.net.eu.org%3E
-  qbe' = qbe.overrideAttrs (_old: {
-    version = "1.1-unstable-2023-08-18";
+  qbe' = qbe.overrideAttrs (old: {
+    version = old.version + "-unstable-2024-01-12";
```

Comment:
i don't think this is right

version should be set based on `rev`. If qbe gets updated to 1.2 but rev is still `852870...` then version should still be 1.1-unstable-2024-01-12

Reply:
I get your point: I've changed this already thinking of dropping it when qbe gets updated to `1.2`, but, as you've pointed out, it looks like it will be updated with every qbe update. Going to change back to the hardcoded usage of `1.1`.

Reply:
Reverted the changes.